src/components/scc/router.h

```diff
@@ -272,6 +286,7 @@ bool router<BUSWIDTH, TARGET_SOCKET_TYPE>::get_direct_mem_ptr(int i, tlm::tlm_ge
     size_t idx = addr_decoder.getEntry(address);
     if(idx == addr_decoder.null_entry) {
         if(default_idx == std::numeric_limits<size_t>::max()) {
+            SCCWARN(SCMOD) << "target address=0x" << std::hex << address << " not found for " << (trans.get_command() == tlm::TLM_READ_COMMAND ? "read" : "write") << " transaction.";
```

Comment:
Same here: hard coding the warning is not a good option. We have use cases where a bus error response is a legal case. This should become configurable.